.github/workflows/scripts/pre-release/references.sh

```diff
@@ -40,6 +40,19 @@ if [[ "$results" != "" ]]; then
     exit 1
 fi
 
+# Verify the internal calls to workflows are referenced by the release tag.
+results=$(
+    find .github/workflows/ -maxdepth 1 -type f -print0 -name '*.yaml' -o -name '*.yml' |
```

Comment:
Do we want this to run on all workflows including pre-submits and such?

Maybe we can exclude workflows that start with `pre-submit.` or `e2e.`?

That would still match the `release.yml` and `scorecards.yml` but maybe we can come up with an update naming scheme for them? Something like `ci.tag.release.yml` and `ci.schedule.scorecards.yml` and then ignore the `ci.` prefix?

Reply:
updated the code to only look for `builder_*` files, as they are the only ones that will use the delegator workflows. Wdut?

Reply:
SG